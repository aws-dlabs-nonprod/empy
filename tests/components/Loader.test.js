import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Loader from '@components/Loader';

describe('Loader', () => {

    test('renders correctly', () => {
        const component = shallow(
            <Loader />
        );

        expect(component).toMatchSnapshot();
    });
});
