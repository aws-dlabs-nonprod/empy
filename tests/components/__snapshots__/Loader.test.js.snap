// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<Loader__StyledLoader>
  <svg
    viewBox="25 25 50 50"
  >
    <circle
      cx="50"
      cy="50"
      r="20"
    />
  </svg>
</Loader__StyledLoader>
`;
